### ISSUE TITLE
Quick Inventory bug: "Special" appears empty after scrolling in "Food"

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#9638]
In the Report a Bug guild, Aerilita ( 3b724a41-48d3-4048-ae27-70b258cf5a72 ) reported:

"On Stable page (website), if Food in Quick Inventory is scrolled (by clicking an arrow at the side), then switching to Special gives me an empty panel (like "second page" of Special, when there's only one page's worth of items). Scared me for a moment when I thought I lost two saddles. Scrolling on the Special tab fixes it (I see them now)."

I've replicated it.

The correct behaviour is that when you switch in either direction between Food and Special in the Quick Inventory, you should be automatically placed in the first "page" for that section.

**Comment by matchajune** [comment]
@Alys May I take this?

**Comment by TheHollidayInn** [comment]
Go for it!

**Comment by lemoness** [comment]
@matchajune This may be related to this problem, since there shouldn't be "pages" at all: https://github.com/HabitRPG/habitica/issues/9366  So you may want to look at both of them at once.

**Comment by matchajune** [comment]
@lemoness I'll grab the other and will do. Thank you!

**Comment by thebat137** [comment]
I've found a possibly related issue in the Food tab of the Quick Inventory.  If I:

1.  Have more than one screen's worth of food (e.g., some of each color),
2.  Scroll away from the beginning of the food list,
3.  Begin feeding pets, and
4.  Run out of enough types of food that I no longer have more than one screen's worth,

Then some of my other types of food disappear from the screen as well and don't reappear until I switch back and forth from the Special tab or refresh the window.  I just did this now and it looks like I only have strawberries, pink cotton candy, and honey, even though, opening the stable page in another window, i can see that I also have meat and potatoes.  I think the food that seems to disappear when I do this is usually the food at the beginning of the list.
